source/portable/mbedtls/core_pkcs11_mbedtls.c

```diff
@@ -5021,7 +5025,7 @@ CK_DECLARE_FUNCTION( CK_RV, C_Verify )( CK_SESSION_HANDLE hSession,
                     }
                 }
 
-                pxSessionObj->xCMACKeyHandle = CK_INVALID_HANDLE;
+                prvCMACCleanUp( pxSessionObj );
```

Comment:
Deserves an update to `CHANGELOG.md` for both this and the previous PR that made memory leak fixes

Reply:
Done.